reports/RJ722.md

```diff
@@ -41,7 +41,7 @@ activity:
  - 1:
    - repo: vulture
    - link: https://github.com/jendrikseipp/vulture/commit/e5012c424fdba1a6aa0b5da8e9befdf2e98e0fc9
-   - details: > 
+   - details: >
```

Comment:
Please don't do unnecessary changes, it's not the scope of this PR.
You can fix this later.